services/README.md

```diff
@@ -1,5 +1,15 @@
 # Preferred Services
 
+## Mailing
+[SendGrid](https://sendgrid.com/), in Heroku: use the free Starter addon.
```

Comment:
I think it we should add it to `app.json`.

Reply:
It's already in there:
https://github.com/abtion/muffi/blob/365ba615a1410094b887ba997852f384e7b5cfcb/app.json#L33-L35
added by:
https://github.com/abtion/muffi/pull/33 👍

Reply:
Added a little note about this: 71f4283e24b472b8916e40b7ceeba48e5f328aa5